test/test_torch_distributed_fsdp_meta.py

```diff
@@ -142,7 +142,8 @@ def meta_module_fn():
 
 def _mp_fn(index):
   device = xm.xla_device()
-  if xm.xla_device_hw(device) in ('TPU', 'GPU'):
+  # This test fails on GPU with 03/30 TF-pin update (https://github.com/pytorch/xla/pull/4840)
```

Comment:
can you include the failing log somewhere in the pr?

Reply:
@ymwangg Heads up we see some regression in some setup of the FSDP on gpu after upadting TF pins, not sure if this will impact you guys.

Reply:
Added!